.travis.yml

```diff
@@ -10,11 +10,12 @@ addons:
       # Needed for NetMHC
       - tcsh
 cache:
-  pip: true
-  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # cache pip files, also directory used for Ensembl downloads of GTF and FASTA files
```

Comment:
Based on this random repo I think you can use multiple `cache` entries: https://github.com/travis-ci/travis-ci/issues/3239